AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Operation/AWSS3StorageUploadFileOperation.swift

```diff
@@ -84,6 +84,18 @@ class AWSS3StorageUploadFileOperation: AmplifyInProcessReportingOperation<
             return
         }
 
+        // If the file does NOT exist, a StorageError.localFileNotFound will be
```

Comment:
can this file get removed after this check? Should we also check for `localFileNotFound` from the S3 response?

Reply:
You mean for us to delete the file represented by `request.local.path`? If so, I would advise against it because the file could be inaccessible but not empty, e.g. iCloud or media library.

Reply:
Discussed offline - resolving...